javascript/javascript1/week1/week1-guzide/ezNamey.js

```diff
@@ -0,0 +1,36 @@
+let firstWords = [
+  "Easy",
+  "Fantastic",
+  "Awesome",
+  "Start",
+  "Dummy",
+  "We-",
+  "Net",
+  "Quick",
+  "Pust-",
+  "Re-",
+];
+let secondWords = [
+  "Corp",
+  "Peasy",
+  "Dum",
+  "Off",
+  "Do-IT",
+  "X",
+  "IT",
+  "Go",
+  "Up",
+  "Hej!",
+];
+
+const randomNumber = Math.floor(Math.random() * 10);
```

Comment:
When we only use one random accessor, we always get the names in parallel, like if the first word is easy, the second one will always be corp. Let's use two random numbers to get truely random names :)